ajax/libs/cheval/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "cheval",
+  "filename": "cheval.min.js",
+  "description": "Copy to the clipboard using JavaScript without writing JS.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ryanpcmcquen/cheval"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com:ryanpcmcquen/cheval.git",
```

Comment:
Let's modify the auto-update config to 
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/ryanpcmcquen/cheval.git",
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "+(cheval|simpleJsCopy)*"
        ]
      }
    ]
  },
```
1. target part cant use git clone to get the files.
2. need to be fixed the Indent.
3. Many I ask about `gCheval.js`@1.2.0? Is this file should be added in CDNJS? Because I didn't find the tutorial said it. Thanks.

Reply:
`gCheval.js` should be included, it is a global variant. That being said I will changing the global version to an `npm` package in the near future.

Reply:
Thanks, Got it, so the auto-update config is 
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/ryanpcmcquen/cheval.git",
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "+(cheval|gCheval|simpleJsCopy)*"
        ]
      }
    ]
  },
```
This should be good enough.
Many thanks.

Reply:
Thank you!